sass/directives/_carousels.scss

```diff
@@ -105,7 +105,7 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  padding: #{$base-margin--small} #{($base-padding / 3)};
```

Comment:
same feedback as for resrouce-continue-block below: since Susan requests specific width and height for the arrow boxes, instead of padding use Susan's numbers and center the arrows inside the boxes. I'm not sure this will work with `display: flex` since there's an inline style `display: inline-block` added so we can work together if you need to to figure out the best way to do this